slc/sample-app/thermostat/efr32/thermostat-thread.slcp

```diff
@@ -179,13 +195,68 @@ configuration:
     condition: [matter_thread_coap_cert_libs]
   - name: SL_STACK_SIZE
     value: "4608"
+    unless: [matter_zigbee_cmp]
+  - name: SL_STACK_SIZE
+    value: "5120"
+    condition: [matter_zigbee_cmp]
   - name: SL_BT_RTOS_EVENT_HANDLER_STACK_SIZE
     value: "1536"
   - name: CIRCULAR_QUEUE_LEN_MAX
     value: 16
   - name: SL_MBEDTLS_PSA_ASSUME_EXCLUSIVE_BUFFERS
     value: 1
 
+# CMP configs for Zigbee Matter
```

Comment:
Configs are always ticky with slcc right? Could we but some in the cmp component you created